Makefile

```diff
@@ -38,24 +40,24 @@ conmon:
 pause:
 	make -C $@
 
-ocid: ${OCID_LINK}
-	go build --tags "$(BUILDTAGS)" -o ocid ./cmd/server/
+ocid: $(wildcard oci/*.go) $(wildcard cmd/server/*.go) $(wildcard server/*.go) ${OCID_LINK}
+	go build --tags "$(BUILDTAGS)" -o $@ ./cmd/server/
 
-ocic: ${OCID_LINK}
-	go build -o ocic ./cmd/client/
+ocic: $(wildcard oci/*.go) $(wildcard cmd/server/*.go) $(wildcard server/*.go)
```

Comment:
We need OCID_LINK in order to build the binaries with the correct vendored dependencies